contents/docs/surveys/webhook.mdx

```diff
@@ -16,20 +16,31 @@ To do this, go to the [data pipeline destinations tab](https://us.posthog.com/pi
 
 2. Make sure the PostHog integration is added to the channel you want to send messages to and select it.
 
-3. Under **Match event and actions**, select **survey sent**. 
+3. Under **Match event and actions**, select **survey sent**.
 
 4. Under **Blocks**, modify the text to include the survey name and response. For example:
 
 ```json
 {
   "text": {
-    "text": "*{person.name}* sent a response to {event.properties.$survey_name}: {event.properties.$survey_response}",
+    "text": "*{person.name}* sent a response to {event.properties.$survey_name}: {event.properties.$survey_response_a3071551-d599-4eeb-9ffe-69e93dc647b6}",
```

Comment:
see https://posthog.slack.com/archives/C07QD3LT8U9/p1743515214079449?thread_ts=1743501911.570249&cid=C07QD3LT8U9
you need to escape things here